plugin/builtin/taskdata/json.go

```diff
@@ -40,6 +40,7 @@ func (hwp *TaskJSONPlugin) GetUIHandler() http.Handler {
 	r.HandleFunc("/task/{task_id}/{name}/tags", uiGetTags)
 	r.HandleFunc("/task/{task_id}/{name}/tag", uiHandleTaskTag).Methods("POST", "DELETE")
 
+	r.HandleFunc("/tags/", uiGetProjectTags)
```

Comment:
why not have this just /tag/ ? 

It also seems preferable/desireable to pass the project explicitly rather than use the cookie.

Reply:
Actually, @john-m-liu asked me to do this. There are some cases when it's acceptable to pass values through context instead of implicit request parms.

Reply:
I felt it made more sense to use the project cookie since this is a UI route where the project should always be the one in the dropdown

Reply:
Yes, it makes sense. I think data like `user`, `locale`, `project`, could accessed through context instead of passing param. I don't know too much about evergreen security, but this approach also forbids making unauthorized request to the endpoint.

Reply:
The context system is a weakness of the UI and increases the overhead of all of our requests (and is the main cause of excess memory use in tasks.) 

I understand if it's not possible, but if there's anyway to pass the project explicitly, I'd prefer a solution like that.